src/tests/stub_libsecurity.cc

```diff
@@ -18,6 +18,14 @@
 CBDATA_NAMESPACED_CLASS_INIT(Security, BlindPeerConnector);
 namespace Security
 {
+BlindPeerConnector::BlindPeerConnector(HttpRequestPointer &, const Comm::ConnectionPointer & aServerConn,
+                                       const AsyncCallback<EncryptorAnswer> & aCallback,
+                                       const AccessLogEntryPointer &alp,
+                                       const time_t) :
```

Comment:
With @rouskov removal of the `const` we must keep the definition and declarations matching.
```suggestion
                                       time_t) :
```